src/main/webapp/queue.js

```diff
@@ -0,0 +1,182 @@
+// Initializes resources for reading/writing to Firestore
+const firebaseConfig = {
+  apiKey: config.apiKey, // eslint-disable-line no-undef
+  authDomain: 'lounge-95f01.firebaseapp.com',
+  databaseURL: 'https://lounge-95f01.firebaseio.com',
+  projectId: 'youtube-lounge',
+  storageBucket: 'youtube-lounge.appspot.com',
+  messagingSenderId: '681171972170',
+  appId: '1:681171972170:web:4c6526b8eb788af9d876b3',
```

Comment:
Is this unique to the app and considered private information?

Reply:
I believe so. I've moved the information into my config file so that it isn't visible on GitHub